### DISCUSSION TITLE
Add influxdb

### DISCUSSION THREAD

**Discussion by peaklabs-dev** [coollabsio/coolify#4038] [Database Requests]
- Add DB Optimized for time-series data and used extensively in IoT projects where you need to store sensor data over time. It's also great for energy-related projects because it's very easy to make calculate power/energy calculations.

**Comment by JuanPablo2655** [comment]
bump, would very much like influxdb as an option

**Reply by danielehrhardt** [reply]
me too or prometheus

**Comment by MikeSoperRubicon** [comment]
Would also find this massively helpful. We currently use influxdb oss unmanaged on vm.

**Comment by nikodunk** [comment]
A possible template to work off here: https://github.com/InfluxCommunity/TIG-Stack-using-InfluxDB-3/blob/main/docker-compose.yml

**Comment by daofficialemissium** [comment]
Bump - for sure this would be great.

**Comment by vinnij** [comment]
Bump - Please

**Comment by danielehrhardt** [comment]
Bump - Please

**Comment by nickcoppins** [comment]
Bump +1

**Comment by gustav-gearloose** [comment]
+ aswell here!

**Comment by eslam5464** [comment]
Bump, this would be very helpfull
